<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemListaMusica</class>
 <widget class="QWidget" name="itemListaMusica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: white;
	border-bottom: 1px solid lightGray;
}
QWidget::hover{
	background-color: rgba(67, 189, 255, 100);
}

QWidget::hover::pressed{
	background-color: rgba(46, 210, 255, 100);
}

QLabel{
	background-color: rgba(0, 0, 0, 0);
	border-style: none;
}

QLabel::hover{
	background-color: rgba(0, 0, 0, 0);
}

QLabel::hover::pressed{
	background-color: rgba(0, 0, 0, 0);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="textoTituloMusica">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="textoArtistaAlbum">
     <property name="text">
      <string>Artist - Album name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="arteAlbum">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="estilos.qrc">:/imgs/np_cp_pad.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="estilos.qrc"/>
 </resources>
 <connections/>
</ui>
